bundles/org.openhab.binding.airgradient/src/main/java/org/openhab/binding/airgradient/internal/handler/MeasureHelper.java

```diff
@@ -0,0 +1,111 @@
+/**
+ * Copyright (c) 2010-2024 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.airgradient.internal.handler;
+
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_ATMP;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_LEDS_MODE;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_PM_003_COUNT;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_PM_01;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_PM_02;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_PM_10;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_RCO2;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_RHUM;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_TVOC;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_UPLOADS_SINCE_BOOT;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.CHANNEL_WIFI;
+import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.PROPERTY_NAME;
```

Comment:
```suggestion
import static org.openhab.binding.airgradient.internal.AirGradientBindingConstants.*
```
(only static imports are allowed with *)

Reply:
Thanks!